proto/agglayer/node/v1/node_state.proto

```diff
@@ -13,6 +14,9 @@ service NodeStateService {
 
   // Method used to get the latest known certificate header for a network.
   rpc GetLatestCertificateHeader(GetLatestCertificateHeaderRequest) returns (GetLatestCertificateHeaderResponse);
+
+  // Method used to get current network status
+  rpc GetNetworkStatus(GetNetworkStatusRequest) returns (GetNetworkStatusResponse);
```

Comment:
I'm wondering if we shouldn't have a `NetworkService` ?

Reply:
Not sure that we do - I think this suits to NodeStateService quite ok.